tests/gnrc_ipv6_ext_opt/main.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2015-2018 Freie Universität Berlin
```

Comment:
```suggestion
 * Copyright (C) 2020 Freie Universität Berlin
```

Reply:
This is basically copied over from `tests/gnrc_ipv6_ext`, so I would like to keep copyright and authorship. I'l update the year range in the copyright note to 2015-2020 though.